### PR TITLE
Improve isNullable/isConstant/isNull/isNotNull performance for LowCardinality argument

### PR DESCRIPTION
**Pull request by Avogar**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Improve isNullable/isConstant/isNull/isNotNull performance for LowCardinality argument

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: tests/queries/0_stateless/02353_isnullable.sql [hunk 1]
```diff
@@ -3,3 +3,9 @@ SELECT isNullable(toNullable(3));
 
 SELECT isNullable(NULL);
 SELECT isNullable(materialize(NULL));
+
```
Comment (devcrafter) [inline tests/queries/0_stateless/02353_isnullable.sql:6]: I found that it's very helpful to use `-- { echoOn }`, - it's much easier to relate output in reference file and queries

**Review by devcrafter — APPROVED** [review]
LGTM with optional comments

File: tests/queries/0_stateless/02353_isnullable.sql (hunk 1 above)
Comment (Avogar) [reply tests/queries/0_stateless/02353_isnullable.sql:6]: Yeah, I know about it, but don't think it's reasonable to use in such small tests

File: tests/performance/low_cardinality_argument.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
         CREATE TABLE test_lc (x UInt64, lc LowCardinality(Nullable(String))) ENGINE = MergeTree order by x
     </create_query>
 
-    <fill_query>INSERT INTO test_lc SELECT number, number % 10 ? NULL : toString(number % 10000) FROM numbers(1000000)</fill_query>
+    <fill_query>INSERT INTO test_lc SELECT number, number % 10 ? NULL : toString(number % 10000) FROM numbers(10000000)</fill_query>
```
Comment (devcrafter) [inline tests/performance/low_cardinality_argument.xml:6]:
```suggestion
    <fill_query>INSERT INTO test_lc SELECT number, number % 10 ? NULL : toString(number % 10000) FROM numbers(1e7)</fill_query>
```

**Pull request merged**